examples/assets/texture/cp437_20x20.ron

```diff
@@ -0,0 +1,1542 @@
+(
+    texture_width: 320,
+    texture_height: 320,
+    sprites: [
+        (
+            x: 0,
```

Comment:
Oh crap, this file is long. Not really a PR to fix this i guess, but I think we should start paying more attention to our asset definition files size. This could easily be serialized in a way to keep a sprite per line or something.

Reply:
I mean yah. Just realistically, this is what real projects spritesheets will contain. Not much we can do about it.